lib/screens/v2/single_alert_widget.ex

```diff
@@ -18,6 +21,7 @@ defprotocol Screens.V2.SingleAlertWidget do
   @doc """
   Gets the Screens.Alerts.Alert struct that this widget contains.
   """
+  # What to change this to?
```

Comment:
I think my comment was "overcome by events" and should have been taken out. 🤔  I was trying to think through whether a function like this would be useful for both a single alert widget and a multi-alert widget (elevatorstatus). Currently, the `alert()` function is only used in BaseAlert in the `location()` function and the `informed_entities` function. I think those functions are best served by still only taking 1 alert and not a collection of alerts, so I'd leave this function as-is.